crate/utils/src/crypto/wrap/encrypt_decrypt.rs

```diff
@@ -100,6 +98,27 @@ where
     }
 }
 
+fn encrypt_with_public_key(
+    wrapping_key: PKey<Public>,
+    plaintext: &[u8],
+) -> Result<Vec<u8>, KmipUtilsError> {
+    // wrap using ECIES/RSA
+    let encrypt_system = HybridEncryptionSystem::new("public_key_uid", wrapping_key);
+    let request = Encrypt {
```

Comment:
Why build an Encrypt request object only to feed it with the plaintext if the method on which it is called only use the plaintext field of the object ?

Same goes for the decrypt version of this method where a Decrypt object is created.